azure-pipelines.yml

```diff
@@ -147,6 +147,16 @@ jobs:
   - script: |
       perl "%DDS_ROOT%\tests\cmake_integration\run_ci_tests.pl" --build-config=$(BuildConfiguration) --generator="Visual Studio 15 2017" --arch $(BuildPlatform)
     displayName: Compile and run CMake tests
+  - script: |
+      cd DevGuideExamples\DCPS\Messenger
+      perl appveyor_run_test.pl
```

Comment:
Consider renaming this perl script.
Should a similar section be added to Linux and macOS configurations?

Reply:
I've added it for both, but the Test fails for MacOS (Example works fine).